core/src/main/java/org/ehcache/spi/cache/Store.java

```diff
@@ -424,7 +424,7 @@
      * @param <V> value type
      * @return the Store honoring the configurations passed in
      */
-    <K, V> Store<K, V> createStore(Configuration<K, V> storeConfig, ServiceConfiguration<?>... serviceConfigs);
+    <K, V> Store<K, V> createStore(ServiceLocator serviceLocator, Configuration<K, V> storeConfig, ServiceConfiguration<?>... serviceConfigs);
```

Comment:
There should not be a need to pass in the serviceLocator here. Upon starting a service receives the `ServiceProvider` it is supposed to use.